### PR TITLE
Info windows2

### PR DESCRIPTION
**Pull request by bgregg**
hopefully finally done. At least for Beta 3. :cry:

### CODE REVIEW

**Comment by bgregg** [comment]
Closes #72.

File: bower.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "localforage": "~1.3.3",
     "underscore": "~1.8.3",
     "angular-underscore-module": "~1.0.3",
-    "angular-resource": "~1.5.0"
+    "angular-resource": "~1.5.0",
+    "js-info-bubble": "*"
```
Comment (akaplo) [inline bower.json]: Can likely remove this

File: www/index.html [hunk 2]
```diff
@@ -31,7 +31,9 @@
 
     <link rel="stylesheet" href="bower_components/ionic-filter-bar/dist/ionic.filter.bar.css">
     <script src="bower_components/ionic-filter-bar/dist/ionic.filter.bar.js"></script>
-    
+    <script src="bower_components/js-info-bubble/src/infobubble.js"></script>
```
Comment (akaplo) [inline www/index.html]: Can likely remove this

File: www/js/services.js [hunk 3]
```diff
@@ -303,19 +303,40 @@ angular.module('pvta.services', ['ngResource'])
...
   function addMapListener(marker, onClick){
     google.maps.event.addListener(marker, 'click', function () {
-            var infoWindow = new google.maps.InfoWindow({
-              content: onClick
-            });
-            infoWindow.open(map, marker);
+      //this auto-closes any bubbles that may already be open
+      //when you open another one, so that only one bubble can
+      //be open at once
+      _.each(bubbles, function(bubble){
```
Comment (akaplo) [inline www/js/services.js]: Since we're not using infobubbles, these variables should likely be renamed.

**Comment by dfaulken** [comment]
Interesting that the CC integration has @werebus's icon

File: www/js/controllers/vehicle-map-controller.js [hunk 4]
```diff
@@ -11,15 +11,26 @@ angular.module('pvta.controllers').controller('VehicleMapController', function (
...
 
 
-  function placeVehicle () {
-    var vehicle = LatLong.getAll();
-    var loc = new google.maps.LatLng(vehicle[0].lat, vehicle[0].long);
-    Map.addMapListener(Map.placeDesiredMarker(loc, 'http://www.google.com/mapfiles/kml/paddle/go.png'), 'Here is your vehicle!');
+  function placeVehicle (vehicle, route) {
+    var vehicleLocation = _.first(LatLong.getAll());
+    var loc = new google.maps.LatLng(vehicleLocation.lat, vehicleLocation.long);
+    var color, message;
```
Comment (dfaulken) [inline www/js/controllers/vehicle-map-controller.js]: Can remove these variables also?

**Comment by akaplo** [comment]
FYI we determined today that the changes this PR makes causes failures in mapping vehicles (completely separate from #116).  Namely, this is caused by the removal of reliance on the LatLong service.

~~Before merging, this needs to get fixed and thoroughly tested.~~
Maybe we should split this into more atomic PRs.

**Pull request merged**